src/Directory.Build.props

```diff
@@ -30,8 +30,8 @@
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2" PublicKey="0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7" />
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2, PublicKeyToken=null" />
     <InternalsVisibleTo Include="ImageSharp.Benchmarks" />
-    <InternalsVisibleTo Include="SixLabors.ImageSharp.Sandbox46" />
     <InternalsVisibleTo Include="SixLabors.ImageSharp.Tests" />
+    <InternalsVisibleTo Include="ImageSharp.Tests.ProfilingSandbox" />
```

Comment:
Isn't this your breaking change?

Reply:
i have added this because i was getting more than 2k errors of `Error	CS0122	'...' is inaccessible due to its protection level`

Reply:
That's because you're including all the files from the test project in the sandbox. 

https://github.com/SixLabors/ImageSharp/pull/1090/files#diff-791cc76ca5e2665ba784771043fd9123R17

We don't do that anymore. The sandbox project simply references the test one.

Reply:
> That's because you're including all the files from the test project in the sandbox.
> 
> https://github.com/SixLabors/ImageSharp/pull/1090/files#diff-791cc76ca5e2665ba784771043fd9123R17
> 
> We don't do that anymore. The sandbox project simply references the test one.

great, thank you! I was not aware of that. I have the removed the tests from the profiling sandbox, now it work works. See: commit: 0bce7af847619aeae7bce0357a1d2179c128cb14